flexget/components/emby/api_emby.py

```diff
@@ -552,9 +552,21 @@ def set_list_search_args(self, args: dict):
             args['SortOrder'] = self.sort['order']
 
         if not self.types or len(self.types) == 0:
-            args['IncludeItemTypes'] = 'Movie;Episode'
+            args['IncludeItemTypes'] = 'Movie,Episode'
         else:
-            args['IncludeItemTypes'] = ';'.join(self.types)
+            args['IncludeItemTypes'] = []
+            for mtyp in self.types:
+                mtyp = str(mtyp).lower
+                if mtyp in ['movie', 'movies']:
+                    args['IncludeItemTypes'].append('Movie')
+                elif mtyp in ['serie', 'series']:
+                    args['IncludeItemTypes'].append('Series')
+                elif mtyp in ['season', 'seasons']:
+                    args['IncludeItemTypes'].append('Season')
+                elif mtyp in ['episode', 'episode']:
+                    args['IncludeItemTypes'].append('Episode')
+
+            args['IncludeItemTypes'] = ','.join(self.types)
```

Comment:
Doesn't seem like we need to support with and without the 's'. Why not just make there be one right way. (When the user is entering it in the config, it's already enforced due to the config validator.)

```suggestion
            args['IncludeItemTypes'] = ';'.join(typ.title() for typ in self.types)
```